model/src/w3wavemd.F90

```diff
@@ -2350,6 +2395,7 @@ SUBROUTINE W3WAVE ( IMOD, ODAT, TEND, STAMP, NO_OUT &
 #endif
 !
                 END IF   !! GTYPE
+!AR: This sections is super dynamic, which means it is not well coded. It changes all the time
```

Comment:
This code comment as written does not contribute to better understanding of the code.  Please revise or remove.

Reply:
Thanks, removed!